src/handlers/counting.js

```diff
@@ -82,13 +82,12 @@ module.exports = async (message, gdb) => {
       countingMessage = await webhook.send(message.content, {
         username: message.author.username,
         avatarURL: message.author.displayAvatarURL({ dynamic: true }),
-      }), {
-      allowedMentions: {
-        users: [],
-        roles: [],
-        parse: [],
-      }
-    });
+        allowed_mentions: {
```

Comment:
![](https://i.imgur.com/Z8FIqAj.png)
d.js allows this option as `allowedMentions` and not `allowed_mentions`. Please update the object accordingly.

Reply:
whoops, didn't notice that

Reply:
Done, thanks :)